generator/src/main/java/com/linecorp/bot/codegen/LineJavaCodegenGenerator.java

```diff
@@ -137,6 +139,72 @@ public ModelsMap postProcessModels(ModelsMap objs) {
         return modelsMap;
     }
 
+    @Override
+    public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs) {
+        super.postProcessAllModels(objs);
+
+        Map<String, ModelsMap> additionalModels = new HashMap<>();
+
+        // List of interfaces for which an Unknown class should not be generated
+        List<String> excluded_interfaces = List.of(
```

Comment:
(nit) In Java, we don't use snake_case
```suggestion
        // List of interfaces for which an Unknown class should not be generated for backward compatibility
        List<String> definedInterfaces = List.of(
```